runhouse/resources/hardware/on_demand_cluster.py

```diff
@@ -464,6 +467,38 @@ def teardown(self):
         Example:
             >>> rh.ondemand_cluster("rh-cpu").teardown()
         """
+        # TODO [SB]: remove the den_auth check once we will get status of clusters without den_auth as well.
+        if self.den_auth:
+            try:
+                # cluster_config = self.config()
+                cluster_status_data = self.status()
+                status_data = {
+                    "status": "terminated",
```

Comment:
do we have an enum for this?

Reply:
We have one on the den side, will add same in OSS as well.